SpectraLogic.SpectraRioBrokerClient.Integration.Test/SpectraRioBrokerClientIntegrationTests.cs

```diff
@@ -129,7 +129,48 @@ public void ArchiveAndRestore()
             }
         }
 
+        [Test, Ignore("https://jira.spectralogic.com/browse/ESCP-371")]
+        public void ArchiveNewFilesOnlyTest_1()
```

Comment:
I know this test is being ignored, but what is it testing? There are no asserts.

Reply:
nm, I get it now

Reply:
👍